vcpkg-registry/ports/cryptoutils/vcpkg.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cryptoutils",
-  "version": "8.2.0",
-  "port-version": 0,
+  "version": "8.3.0",
```

Comment:
Do we need to update this entry manually every time there's a new Zilliqa release? If so, then I'll need to add that to the release step-by-step.

Reply:
No; this port actually shouldn't have existed previously... and wasn't used.